Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/SpeedControlled.mo

```diff
@@ -0,0 +1,89 @@
+within Buildings.Fluid.HeatExchangers.ThermalWheels.Latent;
+model SpeedControlled
+  "Enthalpy recovery wheel with a variable speed drive"
+  extends
+    Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.PartialWheel;
+  parameter Real a[:] = {1}
+    "Coefficients for power consumption curve for rotor. The sum of the elements must be equal to 1"
+    annotation (Dialog(group="Efficiency"));
```

Comment:
Consider adding a table instead of using a polynomial, and then use cubic spline.
This would avoid users having to do a polynomial fit.
Perhaps default motor curves could be proposed as are for fans.